.example-job-board-register-sardonyx.yml

```diff
@@ -27,24 +27,26 @@ features:
 - disabled-ipv6
 - docker
 - docker-compose
-- elasticsearch
+# FIXME: uncomment once working on xenial
```

Comment:
Are there GitHub issues that could be linked on any of these `FIXME`s?

Reply:
There is now!  https://github.com/travis-ci/packer-templates/issues/599